packages/open-payments/src/client/token.test.ts

```diff
@@ -85,4 +140,55 @@ describe('token', (): void => {
       scope.done()
     })
   })
+
+  describe('revokeToken', (): void => {
+    test('returns accessToken if passes validation', async (): Promise<void> => {
+      const accessToken = mockAccessToken()
+
+      const manageUrl = new URL(accessToken.access_token.manage)
+      const scope = nock(manageUrl.origin)
+        .delete(manageUrl.pathname)
+        .reply(200, accessToken)
```

Comment:
```suggestion
        .reply(204)
```
status 204 is the one that gets returned if successful, the `acessToken` isn't returned